### ISSUE TITLE
Can't find yamls/main/modernbert-base.yaml in repo

### ISSUE THREAD

**Issue by KennethEnevoldsen** [AnswerDotAI/ModernBERT#192]
In the readme you refer to `yamls/main/modernbert-base.yaml` however I can't seem to find it in yamls folder.

**Comment by lgienapp** [comment]
Its on an unmerged feature branch still: 

https://github.com/AnswerDotAI/ModernBERT/blob/pretraining_documentation/yamls/main/modernbert-base.yaml

**Comment by NohTow** [comment]
Thanks @lgienapp, this should indeed be the config to use for warmup+constant phases of the pre-training.
Again, I am deeply sorry about the documentation and configs not being cleaner for now, we are working on it but we are all very busy lately...
Will try to bump that so we actually have something cleaner, but the comment linked should allow to experiment for now.
